<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ALSFD</class>
 <widget class="QMainWindow" name="ALSFD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ALSFD</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="conectarBoton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblImagen">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>540</width>
      <height>380</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="alsfd.qrc">:/imagenes/imagenes/label.jpg</pixmap>
    </property>
   </widget>
   <widget class="QListWidget" name="listaEventos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="alsfd.qrc">:/imagenes/imagenes/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="reload">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="alsfd.qrc">
      <normaloff>:/imagenes/imagenes/reload.png</normaloff>:/imagenes/imagenes/reload.png</iconset>
    </property>
    <property name="shortcut">
     <string>F5</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="desconectarBoton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>89</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Desconectar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="puertoSelector">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="alsfd.qrc"/>
 </resources>
 <connections/>
</ui>
